<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Profile Changer</string>
        </property>
        <widget class="QWidget" name="centralWidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QLineEdit" name="settingsFilePath"/>
                </item>
                <item>
                    <widget class="QPushButton" name="selectSettingsFileButton">
                        <property name="text">
                            <string>Select Settings File</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QLineEdit" name="executableFilePath"/>
                </item>
                <item>
                    <widget class="QPushButton" name="selectExecutableButton">
                        <property name="text">
                            <string>Select Executable</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QListWidget" name="profileListWidget"/>
                </item>
                <item>
                    <widget class="QPushButton" name="launchButton">
                        <property name="text">
                            <string>Launch</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="updateButton">
                        <property name="text">
                            <string>Update</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>21</height>
                </rect>
            </property>
            <widget class="QMenu" name="fileMenu">
                <property name="title">
                    <string>File</string>
                </property>
            </widget>
            <widget class="QMenu" name="settingsMenu">
                <property name="title">
                    <string>Settings</string>
                </property>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
